java/parser/src/main/java/org/sqlflow/parser/parse/ParseResult.java

```diff
@@ -0,0 +1,14 @@
+package org.sqlflow.parser.parse;
+
+import java.util.ArrayList;
+
+// ParseResult contains the parsing result of ParseAndSplit
+public class ParseResult {
+  // SQL statements accepted by the parser
+  public ArrayList<String> Statements;
```

Comment:
Change `Statements` to `statements` and remove the `Public`.
Access the class member by `getStatements()/setStatements()` methods.

Reply:
[I searched around](https://stackoverflow.com/questions/11071407/advantage-of-set-and-get-methods-vs-public-variable), looks like getters/setters are recommended developing more complicated classes. Here `ParseResult` is a simple struct, it doesn't have any method and it won't be inherited by any other class.